library/src/main/java/com/getstream/sdk/chat/adapter/MediaAttachmentSelectedAdapter.java

```diff
@@ -81,19 +81,13 @@ public void bind(AttachmentMetaData attachment, final OnAttachmentCancelListener
                     .cornerRadii(cornerRadius, cornerRadius, cornerRadius, cornerRadius)
                     .build());
 
-            if (attachment.file != null && attachment.file.getPath() != null) {
-                File file = new File(attachment.file.getPath());
-                if (file.exists()) {
-                    Uri imageUri = Uri.fromFile(file);
-                    Glide.with(context)
-                            .load(imageUri)
-                            .into(binding.ivMedia);
-                }
-            } else if (attachment.isUploaded()) {
+            if (attachment.file != null) {
+                Glide.with(context)
+                        .load(Uri.fromFile(attachment.file))
```

Comment:
Why are we using `attachment.uri` into `MediaAttachmentAdapter` but we are building the uri by ourselve here?
Could we use the same method in both part of the code?

Reply:
My bad, I've used `attachment.file` only for captured images/videos, but I think it can be simplified